fio/src/fiodriver/fioman.c

```diff
@@ -631,6 +631,17 @@ fioman_remove_frame
     if (p_tx_elem->fiod.fiod == p_sys_fiod->fiod.fiod) {
       /* Does the frame number match one requested? */
       if (FIOMSG_PAYLOAD(p_tx_elem)->frame_no == frame_no) {
+		if (frame_no == FIOMAN_FRAME_NO_51) {
```

Comment:
@gallagjm So you're saying this should be FIOMAN_FRAME_NO_179 here and on line 640?

Reply:
or just on line 640?

Reply:
We should do this generically, check if the tx frame schedule is <FIO_HZ_1 and update the rx_frame count.